packages/babel-parser/src/plugins/flow.js

```diff
@@ -96,7 +96,7 @@ export default (superClass: Class<Parser>): Class<Parser> =>
     }
 
     finishToken(type: TokenType, val: any): void {
-      if (!(type === tt.string || type === tt.semi)) {
+      if (![tt.string, tt.semi, tt.interpreterDirective].includes(type)) {
```

Comment:
I'd prefer that we stick with `===` since this function is called very often and making the temporary array and iterating could be slower.